.local/run-all-tests.sh

```
@@ -0,0 +1,202 @@
+# Stage 5 (optional) — quick 16x sweep + render
+stage_sweep_smoke() {
+  echo "(running 16x sweep, MEASURE_S=60, ~1.5 min)"
+  WARMUP_S=15 MEASURE_S=60 ./protocol-sweep.sh 16 >/tmp/run-all-tests-sweep.log 2>&1
+  local rc=$?
+  if [ "$rc" -ne 0 ]; then
+    echo "  ✗ sweep exited with rc=$rc"
+    tail -10 /tmp/run-all-tests-sweep.log | sed 's/^/    /'
+    return $rc
+  fi
+  local sweep_dir
+  sweep_dir=$(ls -dt /tmp/proto-sweep-2* | head -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)